tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py

```diff
@@ -56,6 +56,16 @@ def run(self):
                             flow_id=self.parameters['flow_id']
                         )
                         return False
+                elif err == "No volumes present in cluster":
```

Comment:
Looks fine but is there better way to find this error, instead of looking of string matching?

Reply:
my point is we can log the error message, it can any error and we can continue import flow Because users can also enable/disable profiling manually.


So we just display an error message in warning and we will continue the import. We don't need to check specific message here.

Reply:
ok